helm/adelphi/tests/kind-config-3-workers.yaml

```diff
@@ -0,0 +1,8 @@
+# 3-worker cluster config
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+- role: control-plane
+- role: worker
+- role: worker
+- role: worker
```

Comment:
This is the Kind cluster configuration for the integration tests, with 3 worker nodes.